From 1217b355af3394b564a8a6704929613d7b5b4f77 Mon Sep 17 00:00:00 2001
From: Karthik Kambatla <kasha@cloudera.com>
Date: Thu, 13 Jun 2013 19:38:40 -0700
Subject: [PATCH 001/596] CLOUDERA-BUILD. Revert "HADOOP-8912. Merging changes
 1397437 and 1397438 from trunk"

This reverts commit 59e991d12389b1a171960bd8c402ba0f89dfa1a0.

Conflicts:
	hadoop-common-project/hadoop-common/CHANGES.txt

Conflicts:
	.gitattributes

(cherry picked from commit e409e38df1b438ea75f5e11a3578ab1c82eef66b)
(cherry picked from commit 6b6c347b0b4f30afa7cbed53ab773bda06ae53d0)
(cherry picked from commit 4c6c1c4edb550ddda8abb6157afda91492b81fa8)
---
 .gitattributes |   20 --------------------
 1 file changed, 20 deletions(-)
 delete mode 100644 .gitattributes

diff --git a/.gitattributes b/.gitattributes
deleted file mode 100644
index 9a4ca70..0000000
--- a/.gitattributes
+++ /dev/null
@@ -1,20 +0,0 @@
-# Auto detect text files and perform LF normalization
-*        text=auto
-
-*.cs     text diff=csharp
-*.java   text diff=java
-*.html   text diff=html
-*.py     text diff=python
-*.pl     text diff=perl
-*.pm     text diff=perl
-*.css    text
-*.js     text
-*.sql    text
-
-*.sh     text eol=lf
-
-*.bat    text eol=crlf
-*.cmd    text eol=crlf
-*.vcxproj text merge=union eol=crlf
-*.csproj text merge=union eol=crlf
-*.sln    text merge=union eol=crlf
-- 
1.7.9.5

